bin/generic-download-tools.sh

```diff
@@ -17,8 +17,8 @@ getContent() {
 # $2 string Destination path to download the file to.
 download() {
 	if [[ `which curl` ]]; then
-		curl -s "$1" > "$2";
+		curl -sL "$1" > "$2";
 	elif [[ `which wget` ]]; then
 		wget -nv -O "$2" "$1"
```

Comment:
```suggestion
		wget -nv -O "$2" "$1"
```
Follows previous suggestion.